kubernetes/loculus/values_preview_server.yaml

```diff
@@ -37,11 +37,11 @@ secrets:
 s3:
   enabled: true
   bucket:
-    endpoint: hel1.your-objectstorage.com
-    region: US
+    region: us-east-1
     bucket: loculus-preview-private
     accessKey: "[[s3accessKey]]"
-    secretKey: "[[s3secretKey]]"      
+    secretKey: "[[s3secretKey]]"
```

Comment:
(these are the only thing I don't really understand – are these placeholders? Or is this the actual value from the sealed secret?)

Reply:
thats placeholders for the preprocessor to replace them from the sealed secrets